Sources/VerticalCardSwiperView.swift

```diff
@@ -34,6 +34,22 @@ public class VerticalCardSwiperView: UICollectionView {
         return (self.isDragging || self.isTracking || self.isDecelerating)
     }
     
+    public override var indexPathsForVisibleItems: [IndexPath] {
```

Comment:
I think I'm going to rename this to indexesForVisibleCards or something like that, that way, users can still access the regular function if they want to do some other stuff with it, and it keeps it easier to use since you really only need the row index and not the segment (since that's always 0).

Reply:
I'm doubting a bit about what I'm going to do with it actually, will probably have to document it somewhere if we go with the different name approach. My goal is to try and keep the project as easy-to-use as possible but also keep the power of `UICollectionView` accessible when you need more than what's supported.

It's also not entirely working if I understand what you're trying to do? 
* When the first card is in focus (index 0), it only returns index 1 (which is the card at the bottom)
* When the card on index 2 or up is focussed then it also returns the 'receded' cards

Unless I'm misunderstanding your implementation?

Reply:
Sounds good. Found a way to classify receded cells yet? I'm currently looking if there's a way to do it in `updateCellAttributes`.

Reply:
Yeah, it's a bit buggy because I haven't found a solid way to get receded cells. I'm currently looking if there's a way to do it in updateCellAttributes.

Reply:
Yeah, that's a bit of a though one.. I'm currently a bit limited on free time but I'll look into it as well whenever I get some free time.

Reply:
@williamsthing I think I have a pretty clean solution (I still need to account for when visibleCells < 3 though)

```swift
public var indexesForVisibleCards: [Int] {
        
        // zPosition : index
        var indexes: [CGFloat: Int] = [:]
        
        for cell in self.visibleCells {
            
            let pos = cell.layer.zPosition
            let index = self.indexPath(for: cell)!.row
            
            indexes.updateValue(index, forKey: pos)
        }
        let lowestPos = indexes.keys.min()
        indexes.removeValue(forKey: lowestPos!)
        
        return Array(indexes.values)
    }
```

Reply:
Sounds good!

Reply:
This is my current fix:
```swift
    public var indexesForVisibleCards: [IndexPath] {
        var paths: [IndexPath] = []
        
        for cell in self.visibleCells {
            if let cell = cell as? CardCell {
                if let indexPath = self.indexPath(for: cell) {
                    if(paths.count > indexPath.row) {
                        paths.insert(indexPath, at: indexPath.row)
                    } else {
                        paths.append(indexPath)
                    }
                }
            }
        }
        
        if(paths.count == 3) {
            paths.remove(at: 0)
        } else if(paths.count == 2) {
            if(paths[0].row > 0) {
                paths.remove(at: 0)
            }
        }
        
        return paths
    }
```

Probably not the cleanest but it works well.

Reply:
I'll look into which implementation I'll go for tomorrow, and I'll look into releasing it then too. 

Thanks again for contributing by the way! 🙂

Reply:
Yeah it's still in beta because it's not 100% stable yet, there is currently one bug i'm still aware off and planning to fix in the future: When swiping away the last card on the stack, the animation glitches out, I'll make an issue on that one to document it too.

Reply:
I think I've finally found a pretty clean way to do it and get correct results. I tested your method but I somehow still got wrong values at specific indexes. This is what I have right now:

```swift
public var indexesForVisibleCards: [Int] {
        
    let lowestIndex = self.indexPathsForVisibleItems.min()?.row ?? 0
        
    // when first card is focussed, return as usual.
    if (visibleCells.count == 2 && lowestIndex == 0) {
        return self.indexPathsForVisibleItems.map({$0.row}).sorted()
    }
        
    var indexes: [Int] = []
    // Add each visible cell except the lowest one and return
    for cell in self.visibleCells {
        
        if let index = self.indexPath(for: cell)?.row, index != lowestIndex {
            indexes.append(index)
        }
    }
    return indexes.sorted()
}
```